.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
The .idea files aren't really portable across users, requiring full paths to things on your machine. Generally you'll just ignore these types of folders with respect to version control since they can be easily re-created by importing the project in the IDE again.